### DISCUSSION TITLE
Add a new English wordlist

### DISCUSSION THREAD

**Discussion by precompute** [monkeytypegame/monkeytype#1293] [Ideas] [answered]
- English (basic) is only good for speed tests.
- English 1K isn't very challenging.
- English 10K is pretty great.
- English 450K is absurdly difficult, filled with medical terms and sometimes incomprehensible words.

I think a "English 50K" wordlist would be a great addition.

I don't know where the current wordlists are sourced from, though.

I found the words in the link below to be approachable and yet more difficult than 10K:
https://en.wiktionary.org/wiki/Wiktionary:Frequency_lists#Project_Gutenberg

**Comment by Miodec** [comment]
The lists are just found on the internet. Maybe the 450k could be slimmed down to 50

**Reply by frogfrogfrogfrogfrog** [reply]
The 450k seems cool to me. Just an absurdly hard word set for those that want to try it. I do think that having an inbetween word set between 10k and 450k is a good idea as well (such as the 50k he's talking about).

**Reply by precompute** [reply]
I agree, 450K shouldn't be removed. 

I'd like to try my hand at making the wordlist, if you're okay with it @Miodec.

**Comment by precompute** [comment, marked as answer]
Has been added:

https://github.com/Miodec/monkeytype/pull/1303
